test/functional-tests/tests/pageActions.js

```diff
@@ -63,3 +63,8 @@ step('Assert page navigated forward', async function() {
   const url = await currentURL();
   assert.equal(new URL(url).pathname, '/checkboxes');
 });
+
+step("Validate part of current url <partOfURL> after redirecting", async function (partOfURL) {
```

Comment:
@Debashis9012 there is validation for the same test 

https://github.com/getgauge/taiko/blob/9f398f194a6056477ef67af672ddbfca204acdb4/test/functional-tests/tests/pageActions.js#L64